packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/internal/RealmSyncUtils.kt

```diff
@@ -63,7 +61,7 @@ internal fun convertSyncErrorCode(error: SyncErrorCode): SyncException {
             // See https://github.com/realm/realm-core/blob/master/src/realm/sync/protocol.hpp#L200
             // Use https://docs.google.com/spreadsheets/d/1SmiRxhFpD1XojqCKC-xAjjV-LKa9azeeWHg-zgr07lE/edit
             // as guide for how to categorize Connection type errors.
-            when (ProtocolConnectionErrorCode.fromInt(error.value)) {
+            when (error.error) {
```

Comment:
Redundant naming?

Reply:
Not really, but we can rename the variable `error: `SyncErrorCode into `syncErrorCode: SyncErrorCode` for readability.